whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmSource.java

```diff
@@ -0,0 +1,9 @@
+package net.ripe.db.nrtm4.persist;
```

Comment:
Can you define a new property instead (or use the existing property `whois.source`)

Reply:
i could, but then there would be only one source -- the story says we need three. is the idea to have three servers, one for each source? i assumed one server for n sources.